Models/Book.cs

```diff
@@ -6,6 +6,9 @@ namespace IndyBooks.Models
     public class Book
     {
         public long Id { get; set; }
+        public string Name { get; set; }
+        public long BookId { get; set; }
+        public long AuthorId { get; set; }
```

Comment:
`BookId` is already defined on line 8.
`AuthorId` should be a `public Writer Author {get;set;}`